src/gui/guiFormSpecMenu.cpp

```diff
@@ -2019,82 +2021,97 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 	{
 		v3f formspec_bgcolor = g_settings->getV3F("formspec_default_bg_color");
 		m_bgcolor = video::SColor(
-			(u8) clamp_u8(g_settings->getS32("formspec_default_bg_opacity")),
-			clamp_u8(myround(formspec_bgcolor.X)),
-			clamp_u8(myround(formspec_bgcolor.Y)),
-			clamp_u8(myround(formspec_bgcolor.Z))
+				(u8) clamp_u8(
+						g_settings->getS32("formspec_default_bg_opacity")),
+				clamp_u8(myround(formspec_bgcolor.X)),
+				clamp_u8(myround(formspec_bgcolor.Y)),
+				clamp_u8(myround(formspec_bgcolor.Z))
```

Comment:
The previous style must be accepted too. Just in case you're trying to satisfy LINT.